www/searchutil.php

```diff
@@ -568,6 +570,52 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
                 }
                 break;
 
+            case 'rounded_median_time_in_minutes':
+                // we need to join the gametimes mv table for this query    
+                if (!isset($extraJoins[$col])) {
+                    $extraJoins[$col] = true;
+                    $tableList .= " inner join gametimes_mv "
+                                  . "on games.id = gametimes_mv.gameid";
```

Comment:
This means that if you search for `minutes:` with no value, no results are returned. I think we might want `LEFT OUTER JOIN` so games without time votes will return in the search results, with `rounded_median_time_in_minutes` set to `NULL`.

Reply:
I changed it to left outer join, but wasn't sure what you meant about setting to null.

Reply:
You did what was needed; this is the effect of `left outer join`.

```
MariaDB [ifdb]> select id, rounded_median_time_in_minutes from games inner join gametimes on games.id=gameid where title = 'Counterfeit Monkey';
Empty set (0.003 sec)

MariaDB [ifdb]> select id, rounded_median_time_in_minutes from games left outer join gametimes on games.id=gameid where title = 'Counterfeit Monkey';
+------------------+--------------------------------+
| id               | rounded_median_time_in_minutes |
+------------------+--------------------------------+
| aearuuxv83plclpl |                           NULL |
+------------------+--------------------------------+
1 row in set (0.015 sec)
```